src/main/kotlin/com/ebay/plugins/metrics/develocity/SummarizerSelectTransform.kt

```diff
@@ -0,0 +1,29 @@
+package com.ebay.plugins.metrics.develocity
+
+import org.gradle.api.artifacts.transform.InputArtifact
+import org.gradle.api.artifacts.transform.TransformAction
+import org.gradle.api.artifacts.transform.TransformOutputs
+import org.gradle.api.file.FileSystemLocation
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
+import org.gradle.work.DisableCachingByDefault
+
+/**
+ * Artifact transform which extracts an individual summarizer's result from the output of the summary
+ * computation task.
+ */
+@DisableCachingByDefault(because = "Copying files does not benefit from caching")
```

Comment:
What about the use case when the file has already been copied?

Reply:
In that case, the transform would be UP-TO-DATE